### ISSUE TITLE
Support robust host services configuration

### ISSUE THREAD

**Issue by autodidaddict** [synadia-io/nex#214]
### Proposed change

Add the following section to the machine configuration JSON file:

```json
{
  ...
 "tags": {
    "demo": "true"
 },
 "host_services": {
    "enabled": true,
    "nats_url": "0.0.0.0:4222",
    "nats_user_jwt": "e...",
    "nats_user_seed": "Sxxx...",
    "kv": {
        "enabled": true,
        "bucket_name": "KV_${workload_name}",
        "max_bytes": 0,
        "jit_provision": false,
    },
    "messaging": {
       "enabled": true,
       "max_request_timeout_ms": 2000,
    },
    "object_store":  {
      "enabled": true,
      "store_name": "BLOB_${workload_name}",
      "max_bytes": 320000,
      "jit_provision": false,
    },
    "http_client": {
       "enabled": true,
       "max_request_timeout_ms": 2000
    }
  }
}
```

Obviously we should be able to add more configuration options as more features are added, but this should set us up nicely to allow someone to operate a secure nexus without worrying that JavaScript functions can exceed their mandate.

### Use case

Currently there's no tangible host services configuration and the host services themselves are tightly coupled (see #170 ) to most of the code. Further, host services currently reuses the same connection as the nex node uses for nexus control, which may not be ideal or desired.

Operators managing a nexus (a cluster of nex nodes) may want to have fine-grained control over the services that are offered to functions and services running within that nexus. For example, they might want to disable some functionality, or disallow just-in-time provisioning, change the NATS connection string, etc.

**Comment by autodidaddict** [comment]
For starters, we should be able to allow the following variable substitutions in the configuration:
* `${workload_name}` - the name of the workload, e.g. `echoservice`
* `${workload_id}` - the unique ID of that workload. Use with caution as this is _per instance_
* `${namespace}` - the namespace in which the workload in question is  operating

**Comment by autodidaddict** [comment]
Closed by #223
